apps/rewards/.eslintrc.js

```diff
@@ -51,6 +52,7 @@ module.exports = {
       }
     ],
     'no-undef': 'error',
+    'no-console': ['warn', { allow: ['warn', 'error'] }],
```

Comment:
I guess is ok to have these additions, but that goes against the good practice of keeping the PR simple, clean and focused, I let you know because is common that you point these kind of things on your reviews @chadoh ! This is not really consistent on your side.

Anyway I am totally ok on keeping these changes, just commenting to know your point about that, if you don't mind.